sdk/ai/azure-ai-projects/1DP_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,365 @@
+# 1DP migration guide.
```

Comment:
Agents migration guide from Hub-based projects to Endpoint-based projects

Reply:
Fixed